setup.py

```diff
@@ -396,6 +396,10 @@ def make_relative_rpath(path):
 # Declare extensions and package
 ################################################################################
 
+# nvcc does not support '-fexceptions'
```

Comment:
this change shouldn't be necessary right? 
the THCUNN extension does not call NVCC. It just autogenerates C++ wrappers and would only call CXX.